doc/unions.adoc

```diff
@@ -182,6 +182,27 @@ function dealWithTree = |elt| -> match {
 }
 ----
 
+=== Default values
+
+Union values members can have default values, in the same way as <<struct-default-values,`struct`>>.
+For instance:
+
+[source,golo]
+----
+union List = {
```

Comment:
Ah got it. It's a documentation bug :-)
When you call `Something.Yum()`, you call the java factory method, which is not redefined by this PR. As for the struct, factory golo functions are generated, so you have to call `Yum` (without prefixing)